src/components/formik-radio/formik-radio.tsx

```diff
@@ -0,0 +1,85 @@
+import { FieldValidator, useField } from 'formik';
+import React from 'react';
+import {
+  FormControl,
+  FormControlProps,
+  FormLabel,
+  FormErrorMessage,
+  VisuallyHidden,
+  Radio,
+  RadioGroup,
+  UseRadioGroupProps,
+  Stack,
+  StackDirection,
+} from '@chakra-ui/react';
+
+interface RadioField {
+  label: string;
+  value: string;
+  disabled: boolean;
+}
```

Comment:
Would it make sense here to extend `RadioProps`?

Reply:
Good point. Done